ftplugin/ps1.vim

```diff
@@ -11,6 +11,8 @@ if exists("b:did_ftplugin") | finish | endif
 " Don't load another plug-in for this buffer
 let b:did_ftplugin = 1
 
+compiler psake
```

Comment:
I think adding the compiler support should be fine, though I'd like @PProvost to weigh in. Part of me feels like it's extraneous to the intent of this bundle.